app-shells/mksh/mksh-9999.ebuild

```diff
@@ -29,14 +29,9 @@ S="${WORKDIR}/${PN}"
 src_compile() {
 	tc-export CC
 	# we want to build static with klibc
-	if use static
-	then
-		unset CC
-		export CC="/usr/bin/klcc"
-		export LDSTATIC="-static"
-	fi
+	if use static; then export CC="/usr/bin/klcc" export LDSTATIC="-static"; fi
 	export CPPFLAGS="${CPPFLAGS} -DMKSH_DEFAULT_PROFILEDIR=\\\"${EPREFIX}/etc\\\""
-	sh Build.sh -r -c lto || die
+	sh Build.sh -r || die
```

Comment:
Can you make it respect `MAKEOPTS`, in particular `-j`?

Reply:
`-j` option is supported by their build script, would you have a recommendation on how to only get this option from `MAKEOPTS` as I’m pretty sure there is conflicting options?

Reply:
Try playing with `-M` first. If that can't be gotten to generate something sane for make to use, multiprocessing.eclass has a helper to extract `-j`.

Reply:
I tried to use `-M` but I failed to use it (and I didn’t find it in Void or NetBSD).
The `-j` flag actually doesn’t accepts any arguments and just allows a for-loop which doesn't wait for the command to finish, which might not be something that we want in gentoo.